source/fundamentals/data-formats/custom-types.txt

```diff
@@ -282,157 +276,145 @@ The ``fallback_encoder`` Callable
-BSON-encodable value. The following fallback encoder encodes python's
-``~decimal.Decimal`` type to a ``~bson.decimal128.Decimal128``:
+BSON-encodable value. 
+
+The following fallback encoder encodes python's ``~decimal.Decimal`` type to a ``~bson.decimal128.Decimal128``:
 
 .. code-block:: python
 
-  >>> def fallback_encoder(value):
-  ...     if isinstance(value, Decimal):
-  ...         return Decimal128(value)
-  ...     return value
-  ...
+   >>> def fallback_encoder(value):
+   ...     if isinstance(value, Decimal):
+   ...         return Decimal128(value)
+   ...     return value
+   ...
 
-After declaring the callback, we must create a type registry and codec options
-with this fallback encoder before it can be used for initializing a collection:
+After declaring the callback, create a type registry and codec options
+with this fallback encoder before initializing a collection:
 
 .. code-block:: python
 
-  >>> type_registry = TypeRegistry(fallback_encoder=fallback_encoder)
-  >>> codec_options = CodecOptions(type_registry=type_registry)
-  >>> collection = db.get_collection("test", codec_options=codec_options)
-  >>> collection.drop()
+   >>> type_registry = TypeRegistry(fallback_encoder=fallback_encoder)
+   >>> codec_options = CodecOptions(type_registry=type_registry)
+   >>> collection = db.get_collection("test", codec_options=codec_options)
+   >>> collection.drop()
 
-We can now seamlessly encode instances of ``~decimal.Decimal``:
+You can now encode instances of ``~decimal.Decimal``:
 
 .. code-block:: python
 
-  >>> collection.insert_one({"num": Decimal("45.321")})
-  InsertOneResult(ObjectId('...'), acknowledged=True)
-  >>> mydoc = collection.find_one()
-  >>> pprint.pprint(mydoc)
-  {'_id': ObjectId('...'), 'num': Decimal128('45.321')}
-
+   >>> collection.insert_one({"num": Decimal("45.321")})
+   InsertOneResult(ObjectId('...'), acknowledged=True)
+   >>> mydoc = collection.find_one()
+   >>> pprint.pprint(mydoc)
+   {'_id': ObjectId('...'), 'num': Decimal128('45.321')}
 
 .. note::
 
-  Fallback encoders are invoked *after* attempts to encode the given value
-  with standard BSON encoders and any configured type encoders have failed.
-  Therefore, in a type registry configured with a type encoder and fallback
-  encoder that both target the same custom type, the behavior specified in
-  the type encoder will prevail.
-
-
-Because fallback encoders don't need to declare the types that they encode
-beforehand, they can be used to support interesting use-cases that cannot be
-serviced by ``TypeEncoder``. One such use-case is described in the next
-section.
-
+   Invoke fallback encoders *after* attempts to encode the particular value
+   with standard BSON encoders and any configured type encoders have failed.
+   Therefore, in a type registry configured with a type encoder and fallback
+   encoder that both target the same custom type, the behavior specified in
+   the type encoder prevails.
```

Comment:
The second sentence starts with "Therefore," which makes me think the first sentence is explaining something, but it's actually imperative ("Invoke fallback encoders...").

Does this suggestion match the meaning of the original?

```suggestion
   {+driver-short+} invokes fallback encoders *after* attempts to encode the particular value
   with standard BSON encoders and any configured type encoders have failed.
   Therefore, in a type registry configured with a type encoder and fallback
   encoder that both target the same custom type, the behavior specified in
   the type encoder prevails.
```

Reply:
Good catch, your suggestion is what it should have been